NickvisionTubeConverter.Shared/Models/Download.cs

```diff
@@ -309,23 +298,22 @@ public void Start(DownloadOptions options)
                             });
                         }
                         PyObject success_code = ytdlp.YoutubeDL(_ytOpt).download(new List<string>() { MediaUrl });
-                        if ((success_code.As<int?>() ?? 1) != 0)
-                        {
-                            Filename = Regex.Unescape(Filename);
-                        }
                         KillAriaKeeper();
                         ForceUpdateLog();
                         IsDone = true;
                         IsRunning = false;
                         _outFile.close();
                         IsSuccess = (success_code.As<int?>() ?? 1) == 0;
+                        if(IsSuccess)
+                        {
+                            File.Move($"{SaveFolder}{Path.DirectorySeparatorChar}{Id.ToString()}{FileType.GetDotExtension()}", $"{SaveFolder}{Path.DirectorySeparatorChar}{Filename}");
```

Comment:
There may be multiple files, not just one, in case subtitles were downloaded but not embedded.

Reply:
Download webm + srt subtitles to reproduce

Reply:
The subtitles filename ends up being the ID as well, correct? I just have to rename that too...

Reply:
> Also, there's no chars replacement anywhere now, right? So the issue with NTFS is not solved.

What do you mean by this? The files are guaranteed to not have any problematic characters since Id is just numbers, letters, and `-`.

Reply:
> There may be multiple files, not just one, in case subtitles were downloaded but not embedded.

Fixed in latest commit

Reply:
> What do you mean by this? The files are guaranteed to not have any problematic characters since Id is just numbers, letters, and -.

The Id is, but not the user-defined file name.

Reply:
I see what you're saying...when the file gets renamed to the user-defined file name and it contains invalid characters it's still not good...

Should we Regex.Escape() the Filename right before renaming as it will take care of the characters not allowed?